python/xorbits/_mars/services/storage/handler.py

```diff
@@ -124,13 +124,18 @@ async def get(
         session_id: str,
         data_key: str,
         conditions: List = None,
+        cpu: Optional[bool] = None,
         error: str = "raise",
     ):
         try:
             data_info = await self._data_manager_ref.get_data_info(
                 session_id, data_key, self._band_name
             )
             data = yield self._get_data(data_info, conditions)
+            from xorbits._mars.dataframe.utils import is_cudf
+
+            if is_cudf(data) and cpu:
+                data = data.to_pandas()
```

Comment:
Looks like no ut covered this line.

Reply:
Yeah, I'm working on new test cases for this scenario.